doc/nrf/protocols/matter/end_product/certification.rst

```diff
@@ -257,10 +257,12 @@ Matter dependent certification inheritance
 If your product uses qualified Bluetooth stack or certified Thread libraries (or both) provided as part of the |NCS|, you can *inherit* certification from Nordic Semiconductor, provided that you do not introduce any changes to these stacks.
 In practice, this means reusing Nordic Semiconductor's certification identifiers, which were obtained as a result of the official certification procedures.
 
+When reusing Nordic Semiconductor's certification identifiers, you still need to join the SDO organization and apply for certification by inheritance.
```

Comment:
This PR was prompted by a customer asking if he really had to join an org, even after reading the docs.
Therefore I think it would be more clear if we spell out "join SDO". Is not all requirements to join orgs anyway?
However, if we expect there to be added non-organization requirements later, I see why we want to be more generic

Reply:
Do you think that we should describe what are the requirements for each SDO to apply for certification?

Reply:
We can't describe the requirements for each SDO, because we would have to start to monitor third-party processes.

Matter does not yet offer certification by inheritance. When it does, we will update this page for sure, but for now we're stuck to the Matter full certification process that can use components for which certification can be inherited, which does not equal full "Matter certification by inheritance".

Speaking of which, shouldn't we link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/protocols/thread/certification.html#thread-certification-options in the Thread section? @doublemis1 ? @LuDuda ?

Reply:
@LuDuda , should we add the link to the Thread section, as per comments above?

Reply:
I think it does not harm to add the link to Thread section. However i would leave the wording about joining the SDO. I agree we should not go into big details, but we are quite well-informed in Thread and BLE SDOs, so that we should be able to catch changes in this area (i.e. if different membership level is required).